### DISCUSSION TITLE
n/a

### DISCUSSION THREAD

**Discussion by adminx123** [community/community#81392] [Other Feature Feedback, Questions, & Ideas]

**Comment by SIGMazer** [comment]
first make sure that you deploy in the correct branch.
second remove the custom domain `inexasli.com` if you do not pay for it and just use default github pages link `inexasli.github.io` and you should found it in deployment like this 
![image](https://github.com/community/community/assets/88988252/459c0b3f-d5fc-41ad-83ab-52690421b445)

**Reply by SIGMazer** [reply]
I think deployment not work because github complaining about your DNS.
For testing try to use default DNS
